bodhi/server/templates/fragments.html

```diff
@@ -7,7 +7,9 @@
           ${util.markup(comment.text) | n}
         </div>
         <div class="col-md-2 text-xs-right text-muted">
-          <a href="${comment.url()}">${util.age(comment.timestamp)}</a>
+          <a href="${comment.url()}" data-toggle="tooltip" title="${comment.timestamp.strftime("%Y-%m-%d %H:%M:%S")}">
```

Comment:
Some of the timestamps in the right hand box specify that the timestamp is UTC in their hovertexts (like the "Submitted" and "In testing" times") - would you like to add that here too?

Weirdly, it seems like the number of days to stable one doesn't have a hoverover.

Anyways, I'm fine with this patch as-is if you like this, but those are just some thoughts. Let me know!

Reply:
Done. Should I add a hovertext to the days-to-stable entry?

Reply:
@elyscape it's up to you - if you want to go for it.

Reply:
Actually, looks like this was done on purpose. `update.days_to_stable` is a property that returns the number of days left until the update is eligible for stable. To add a tooltip to it, I'd either need to use that to calculate the time until stable in the template or add another property to the `Update` class that returns that value. As such, I'll leave it out, and we can consider adding it in a separate pull request.